Framework/DataHandling/test/LoadNexusLogsTest.h

```diff
@@ -168,6 +169,16 @@ class LoadNexusLogsTest : public CxxTest::TestSuite {
                                           periodValues.end());
     TSM_ASSERT_EQUALS("Should have 4 periods in total", 4,
                       uniquePeriods.size());
+
+    const bool hasProtonChargeByPeriod = run.hasProperty("run_title");
```

Comment:
That should be `proton_charge_by_period`. Since we are using that below anyway, I will remove it.